packages/react-radio/package.json

```diff
@@ -42,6 +42,7 @@
     "react-test-renderer": "^16.3.0"
   },
   "dependencies": {
+    "@fluentui/react-icons": "^2.0.159-beta.10",
     "@fluentui/react-utilities": "9.0.0-rc.3",
```

Comment:
nit: We should move this after `fluentui/react-theme` to keep these alphabetically ordered.